run_blockchain_terraformer.py

```diff
@@ -715,7 +715,7 @@ def organize_pool_details_solidly_v2(
 
     stable_pool = "stable" if pool_data["args"]["stable"] else "volatile"
     pool_contract = async_web3.eth.contract(address=pool_address, abi=exchange_object.get_abi())
-    fee_float = asyncio.get_event_loop().run_until_complete(asyncio.gather(exchange_object.get_fee(address=pool_address, contract=pool_contract)))
+    fee_str, fee_float = asyncio.get_event_loop().run_until_complete(asyncio.gather(exchange_object.get_fee(address=pool_address, contract=pool_contract)))[0]
```

Comment:
How come this change works?
What did the call to function `run_until_complete` return before this change, and what does it return after?

Reply:
Expected return before was a single float.

After recent changes, however, it now returns List[Tuple[str, float]]

This simply takes the first and only item in the List and unpacks the tuple into str, float.